.devops/test-deploy-setup-client-sdk.yml

```
@@ -71,7 +71,7 @@ stages:
   # Template for generating and deploying client SDk to NPM
   - template: azure-templates/deploy-client-sdk.yml
     parameters:
-      # dependsOn: [Release]
+      dependsOn: [Release]
```

I may miss something on yaml syntax, but shouldn't it be

```suggestion
      dependsOn:
        - Release
```

?

---

Maybe we can use both, don't know.. [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/stages?view=azure-devops&tabs=yaml) it says ` dependsOn: string | [ string ]` and to be sure I made a small test [here](https://dev.azure.com/pagopa-io/io-functions-test-deploy/_build/results?buildId=9834&view=results) with two dependencies